plugins/nf-ga4gh/src/main/nextflow/ga4gh/tes/client/ApiClient.java

```diff
@@ -68,7 +68,7 @@
 
 public class ApiClient {
 
-    private String basePath = "http://localhost";
+    private String basePath = "/ga4gh/tes/v1";
```

Comment:
is this correct?  typically TES has a url like: `https://tes.azurewebsites.net/v1/tasks`

Reply:
Correct. This does get over written by the config. I changes from default of localhost to `/ga4gh/tes/v1` to share what the spec has an example url. I can change it back it has no implication on the code.